cluster/examples/kubernetes/nfs/crd.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: export.nfs.rook.io
+spec:
+  group: nfs.rook.io
+  names:
+    kind: NFSExport
```

Comment:
the kind should be the capitalized singular form of the metadata.name field.  E.g., `Export`.

Reply:
I will change the metadata.name field to `nfsexports.nfs.rook.io`